gax-java/gax-httpjson/src/main/java/com/google/api/gax/httpjson/ProtoMessageRequestFormatter.java

```diff
@@ -147,6 +147,12 @@ public Builder<RequestT> setAdditionalPaths(String... rawAdditionalPaths) {
       return this;
     }
 
+    @InternalApi
+    public Builder<RequestT> updateRawPath(String rawPath) {
```

Comment:
Is it possible to initialize the `ProtoMessageRequestFormatter` with the updated path instead of overriding it later?